<launch>
	<param name="joy/deadzone" value="0.33" />
	<node name="joy" pkg="joy" type="joy_node" output="screen" />
	<node name="ssl_robot_joy" pkg="ssl_robot_comm" type="joy2vel_xyw_smooth" output="screen" />
	
	<param name="vDef" value="1" />
	<param name="vMax" value="3" />
	<param name="dccDef" value="2" />
	<param name="dccMax" value="4" />
	<param name="accDef" value="2" />
	<param name="accMax" value="3" />
	<param name="rotateDef" value="5" />
	<param name="rotateMax" value="10" />
	<param name="rotateMin" value="1" />
	<param name="affw" value="false" />
	
    <node name="ssl_robot_control" pkg="ssl_robot_control" type="vel_xyw2motorCtrl" output="screen" />
</launch>